rollingpin/commands.py

```diff
@@ -0,0 +1,82 @@
+from abc import ABCMeta, abstractproperty
+
+
+class Command(object):
+    __metaclass__ = ABCMeta
+
+    CONTINUE = 1
```

Comment:
if you wanna pull in the `enum34` library you could define this as a proper enumeration https://docs.python.org/3/library/enum.html

probably not worth it though